icons/pause.svg

```diff
@@ -9,6 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <rect width="4" height="16" x="6" y="4" />
-  <rect width="4" height="16" x="14" y="4" />
+  <rect width="4" height="18" x="6" y="3" rx="1" />
+  <rect width="4" height="18" x="14" y="3" rx="1" />
```

Comment:
Not sure about increasing the height of this one, can we create side by side comparison? To me the pause icon looks a bit less balanced on 100% scale.

Reply:
Yeah, I think this also needs an increase in width to match the increase in height (see my original images).

Reply:
> can we create side by side comparison?

<img width="264" alt="Untitled" src="https://user-images.githubusercontent.com/7797479/234373406-84b6a42b-4f92-4a4c-ade0-0010a8bfd8fc.png">

<img width="90" alt="Untitled" src="https://user-images.githubusercontent.com/7797479/234372942-3ac54e00-684c-4e20-afe8-d82418d4666f.png">

> I think this would also need an increase in width to match the increase in height

See first row…

Reply:
Yeah, I think out of the three, the first row matches `square` the best.

Reply:
@danielbayley Thanks. I checked it with other player control icons. And the first one looks the best, I think.

Reply:
@ericfennis @karsa-mistmere Yep, that version was `push`ed…